openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -307,8 +360,10 @@ function initWebSockets($) {
                                 </div>
                            </div>`);
             $('.ow-notification-toast-wrapper').prepend(toast);
-            notificationSound.currentTime = 0;
-            notificationSound.play();
+            if (notificationWindow.canPlaySound()){
```

Comment:
It will be better to add a comment here mention the reason why this condition is being checked here.